### PR TITLE
fix contentview cli failures

### PR DESCRIPTION
**Pull request by latran**
It's not possible to add an rpm to the the /var/lib/pulp/published path anymore due to pulp3.  Rewriting test case another way.

### CODE REVIEW

**Comment by latran** [comment]
trigger: test-robottelo
pytest: pytest tests/foreman/ui/test_contentview.py::test_positive_composite_child_inc_update

**Comment by latran** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py::test_positive_composite_child_inc_update

**Comment by latran** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py::test_positive_composite_child_inc_update

**Comment by latran** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py::test_positive_composite_child_inc_update
pytest: tests/foreman/ui/test_contentview.py::test_positive_errata_inc_update_list_package

**Comment by latran** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py::test_positive_composite_child_inc_update

**Comment by latran** [comment]
results:
```
pytest tests/foreman/ui/test_contentview.py::test_positive_errata_inc_update_list_package

======= 1 passed, 3 warnings in 288.33s (0:04:48) ==========

pytest tests/foreman/ui/test_contentview.py::test_positive_composite_child_inc_update

========== 1 passed, 3 warnings in 1199.58s (0:19:59) ===========
```

File: tests/foreman/ui/test_contentview.py [hunk 1]
```diff
@@ -3089,56 +3071,78 @@ def test_positive_composite_child_inc_update(session, rhel7_contenthost, default
...
+        distro=constants.DISTRO_RHEL7, repositories=[SatelliteToolsRepository()]
     )
     content_data = repos_collection.setup_content(org.id, lce.id, upload_manifest=True)
+    # adding custom repo subscription to ak
+    ak_id = content_data['activation_key']['id']
+    command = (
+        f'hammer activation-key add-subscription --id {ak_id} '
+        f'--subscription {product.name} --organization-id {org.id}'
+    )
+    result = default_sat.execute(command)
```
Comment (jyejare) [inline tests/foreman/ui/test_contentview.py:3110]: You could have used nailgun/API for faster performance https://github.com/SatelliteQE/nailgun/blob/master/nailgun/entities.py#L282 

Non-blocking comment though !

**Review by jyejare — APPROVED** [review]
ACK with non-blocking comments !

**Review by pondrejk — APPROVED** [review]

**Pull request merged**